frontend/src/views/CellGuide/components/common/OntologyDagView/components/AnimatedNodes/index.tsx

```diff
@@ -38,6 +38,7 @@ interface AnimatedNodesProps {
   hideTooltip: () => void;
   cellTypesWithMarkerGeneStats: MarkerGeneStatsByCellType | null;
   setCellInfoCellType?: (props: CellType | null) => void;
+  setLastNodeClicked: (nodeId: string) => void;
```

Comment:
This will control which nodes get centered after expanding/collapsing a node in the DAG.